app/lib/actions/katello/host/package/update.rb

```diff
@@ -2,18 +2,13 @@ module Actions
   module Katello
     module Host
       module Package
-        class Update < Actions::EntryAction
-          include Helpers::Presenter
-
-          def plan(host, packages)
-            Type! host, ::Host::Managed
+        class Update < Actions::Katello::AgentAction
+          def self.agent_message
+            :update_package
+          end
 
-            action_subject(host, :hostname => host.name, :packages => packages)
-            plan_action(Pulp::Consumer::ContentUpdate,
-                        consumer_uuid: host.content_facet.uuid,
-                        type:          'rpm',
-                        args:          packages)
-            plan_self(:host_id => host.id)
+          def agent_action_type
+            :content_install
```

Comment:
should this be ```:content_update```

Reply:
I don't think so. See the usage in DispatchHistoryPresenter and compare to behavior in the current presenter